projects/aca-content/src/lib/components/recent-files/recent-files.component.scss

```diff
@@ -0,0 +1,9 @@
+aca-recent-files {
+  .aca-header-container {
```

Comment:
this is a repetitive class, could be placed into a single place somewhere

Reply:
done